app/helpers/session_timeout_warning_helper.rb

```diff
@@ -18,7 +18,7 @@ def expires_at
   def timeout_refresh_path
     UriService.add_params(
       request.original_fullpath,
-      timeout: true,
+      timeout: :form,
     )&.html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
Do we still need the `html_safe` on there? It looks like the only use of this method is for a `data-` attribute which I expect would be "smarter" about HTML escaping already?

Reply:
I'll check this on Monday, this might be one of the things we could remove altogether as a post-deploy cleanup, once we move everything off the current session API endpoints.

I should probably get that follow-on pull request prepped anyways.

Reply:
To this and the prior comment, I think I might leave this alone for now. It's still needed, but ties into some bigger problems with the auto-refresh that I'd like to sort out ([related Slack discussion](https://gsa-tts.slack.com/archives/C0NGESUN5/p1678739597630529)). At the very least, it might make sense to just have the JavaScript be responsible for navigating to `?timeout=form`, rather than passing a full URL path from the server.